### PR TITLE
Fix the mobile styling for the orders table.

### PR DESCRIPTION
**Pull request by Camwyn**
### Fixes:
 - use our custom props!
 - hide the gateway column (on mobile)
 - add some padding to the expanded row (mobile)
 - tweak the spacing

### Also:
 - Fix the .puprc workflow issue
 - convert uses of "start" and "end" to flex-start" and "flex-end" for better support

### Artifacts:
Desktop:
Screenshot 2024-07-10 at 5 34 52 PM
Mobile:
Screenshot 2024-07-10 at 5 35 21 PM

### Ticket
[ET-2147]

[ET-2147]: https://stellarwp.atlassian.net/browse/ET-2147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Commerce/Admin_Tables/Orders_Table.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ public function search_box( $text, $input_id ) {
 
 		$text = __( 'Search Orders', 'event-tickets' );
 		?>
-			<p class="search-box">
+			<div class="search-box">
```
Comment (dpanta94) [inline src/Tickets/Commerce/Admin_Tables/Orders_Table.php:86]: `<p>` replacement to `<div>` without amending closing tag

File: src/Tickets/Commerce/Admin_Tables/Orders_Table.php [hunk 2]
```diff
@@ -110,7 +110,10 @@ public function search_box( $text, $input_id ) {
 	 * @return string The HTML for the row actions.
 	 */
 	protected function row_actions( $actions, $always_visible = false ) {
-		return '';
+		return '<button type="button" class="toggle-row"><span class="screen-reader-text">' .
+			/* translators: Hidden accessibility text. */
+			__( 'Show more details' ) .
```
Comment (dpanta94) [inline src/Tickets/Commerce/Admin_Tables/Orders_Table.php:115]:
```suggestion
			__( 'Show more details', 'event-tickets' ) .
```

File: src/Tickets/Commerce/Admin_Tables/Orders_Table.php [hunk 3]
```diff
@@ -476,25 +479,25 @@ public function column_total( $item ) {
...
+				'<div class="tec-tickets-commerce-price-container"><ins><span class="tec-tickets-commerce-price">%s</span></ins></p>',
 				$item->total_value->get_currency()
 			);
 		}
 
 		if ( empty( $item->gateway_payload['refunded'] ) ) {
 			// The item was refunded but we don't know anything about it.
 			return sprintf(
-				'<p class="tec-tickets-commerce-price-container"><ins><span class="tec-tickets-commerce-price">%s</span></ins></p>',
+				'<div class="tec-tickets-commerce-price-container"><ins><span class="tec-tickets-commerce-price">%s</span></ins></p>',
```
Comment (dpanta94) [inline src/Tickets/Commerce/Admin_Tables/Orders_Table.php:500]: `<p>` to `<div>` without amending closing tags

File: src/Tickets/Commerce/Admin_Tables/Orders_Table.php [hunk 4]
```diff
@@ -506,7 +509,7 @@ public function column_total( $item ) {
 		$total_value = $total - $refunded;
 
 		return sprintf(
-			'<p class="tec-tickets-commerce-price-container"><ins><span class="tec-tickets-commerce-price">%s</span></ins><del><span class="tec-tickets-commerce-price">%s</span></del></p>',
+			'<div class="tec-tickets-commerce-price-container"><ins><span class="tec-tickets-commerce-price">%s</span></ins><del><span class="tec-tickets-commerce-price">%s</span></del></p>',
```
Comment (dpanta94) [inline src/Tickets/Commerce/Admin_Tables/Orders_Table.php:512]: Same as above

**Review by dpanta94 — CHANGES_REQUESTED** [review]
Left a couple of Small Fixes @Camwyn 🚀

File: src/Tickets/Commerce/Admin_Tables/Orders_Table.php (hunk 1 above)
Comment (Camwyn) [reply src/Tickets/Commerce/Admin_Tables/Orders_Table.php:86]: 🤦🏼

**Review by dpanta94 — COMMENTED** [review]
Looks good @Camwyn ! Just needs some integration updates on tests to keep them from failing. Should be simply updating the snapshots

**Review by dpanta94 — APPROVED** [review]

**Pull request merged**